teslajsonpy/controller.py

```diff
@@ -1146,11 +1045,6 @@ def charging_state(self, car_id: Text = None, vin: Text = None) -> Text:
             return self.get_charging_params(vin=vin).get("charging_state")
         return None
 
-    def get_power_params(self, site_id: Text) -> Dict:
-        """Return cached copy of power_params for site_id."""
-        energysite_id = self._id_to_energysiteid(site_id)
-        return self.__power[energysite_id]
-
     def get_state_params(self, car_id: Text = None, vin: Text = None) -> Dict:
```

Comment:
What was the reasoning for being able to get car parameters using `car_id`? With how I'm restructuring this rewrite, I'm just storing the entire `VEHICLE_DATA` response to `self._vehicle_data` by VIN and passing that to the new `TeslaCar` class when it's instantiated. That makes most of these `get_*` and `set_*` methods not needed anymore but I just want to make sure I'm understanding what the purpose was for being able to use `car_id` or `vin` to get and set car data.

Reply:
We allowed getting a single cars info instead of multiple.

Reply:
You can do that with the VIN though and everywhere I see these methods being called, it's used with the vin argument.

Reply:
You'd have to check the history. The vin may not have been an option when we wrote it.